woocommerce/woocommerce-gutenberg-products-block
# Products by Rating: Add block transform for the legacy rating widget

Fixes #7173

This PR provides a block transform for the legacy rating widget. 

|Before|After|
|-|-|
|![before_ratings](https://user-images.githubusercontent.com/905781/194530578-feecdaca-a77d-4865-9cef-9dd16380df7d.jpg)|![after_ratings](https://user-images.githubusercontent.com/905781/194530677-30b30cf3-7b74-4e2e-ad8f-41dd91fb1bbb.jpg)|

**Prerequisites:** 

In order to test this, you'll need to add the above-mentioned legacy widget in the Widgets screen. This PR is also hiding that widget as an option for the user (as we want them to use the blocks instead). The easiest way for you to make them show up for testing again is probably to just remove the following line in `woocommerce-gutenberg-products-block`:

<img width="859" alt="Screenshot 2021-05-31 at 11 47 30" src="https://user-images.githubusercontent.com/1562646/120176730-ece62f00-c207-11eb-9d5f-d68642572aae.png">

**What to test:**

1. Go to the Widgets editor in Appearance > Widgets.
2. Add the Filter Products by Rating widget.
3. Transform the widget using the provided block transform option. 
4. Confirm the transformation works as expected and retains changed options that have an equivalent in the blocks.
 

### WooCommerce Visibility





### Changelog

> Filter by rating: Provide a block transform for the legacy rating widget
